src/OrchardCore.Themes/TheAgencyTheme/Assets/TheAgencyTheme/dist/js/scripts.js

```diff
@@ -1,11 +1,11 @@
 /*!
 * Start Bootstrap - Agency v7.0.11 (https://startbootstrap.com/theme/agency)
-* Copyright 2013-2022 Start Bootstrap
```

Comment:
I think such changes need to be done on the theme repo itself

Reply:
If I recall, there was an old change requested a long time ago, and @Piedone asked it to be done on the original repo. 

@Piedone correct me if I'm wrong